great-idea-website/css/index.css

```diff
@@ -63,4 +63,206 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+body {
+	width: 950px;
+}
+a {
+	text-decoration: none;
+}
+
+.container {
+	width: 950px;
+	margin-bottom: 40px; 
+	padding-bottom: 90px; 
+	/* border-bottom: 2px solid black; */
```

Comment:
I liked the fact that you commented out code to check it but I would like to see you go back and fully remove them to clean your code up a bit. This makes your code more readable in production where there are a lot of lines.

Reply:
Done. Tnx for a comment, really appreciate it.